scripts/spack/packages/conduit/package.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
I copied this over from the conduit recipe on spack, and kept the relevant Axom-related edits (marked by `AXOM EDIT START/END` comments).

See: https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/conduit/package.py